erpnext_com/www/docs/user/manual/en/setting-up/print/raw-printing.md

```diff
@@ -0,0 +1,93 @@
+# Raw Printing
+
+> Introduced in Version 12
+
+### 1. Introduction to Raw Printing
+
+Sending a string of commands to a printer directly in its native language is called Raw Printing. Many thermal printers need these raw commands sent to them in order to perform functions like barcode printing, receipt printing, label printing, etc. Raw Printing bypasses the printer's drivers in most cases, making it very fast and reliable. Raw Printing is also capable of doing some advanced features such as cutting receipt paper, kicking out cash drawers, etc.
+
+### 2. Setting up Raw Printing in ERPNext
+
+Below are three simple steps to get started with Raw Printing:
+
+#### 2.1 Installing QZ Tray application on the client computer
+
+Download and install the QZ Tray application on the computer to which your thermal printer is connected. This application can be found at its [official site](https://qz.io/download/). Currently, Windows, macOS, and Linux are supported by QZ Tray. During the installation you will be prompted to install Java if not already installed, please install Java to complete the installation.
+Further instructions on installing the QZ Tray Application can be found at the below link:
+- [Installing QZ Tray](https://qz.io/wiki/using-qz-tray)
+
+#### 2.2 Create Raw Commands Print Format
+
+To be able to send raw commands to a printer you need to first create a print format in raw commands. Jinja Templating Language is used in raw commands just like in the [HTML custom print format](/docs/user/manual/en/customize-erpnext/print-format).
+To create a new print format for raw printing:
+1. Go to print format list: **Setting > Printing > Print Format**
+2. Create a new print format document
+3. Select the relevant DocType
+4. Check the `custom format` and `Raw Printing` options
+5. Fill in the `Raw Commands` field with the required raw commands to be sent to the printer.
+6. Click save. 
+
+![Raw Commands Print Format]({{docs_base_url}}/assets/img/setup/print/raw-command-print-format.png)
+
+Currently, any string-based printer languages can be used in the `Raw Commands` field in the print format. Writing raw commands requires knowledge of the printer's native language provided by the printer manufacturer. Please refer to the developer manual provided by the printer manufacturer on how to write their native commands.
+
+#### 2.3 Enable Raw Printing in the Print Setting
+
+To enable raw printing:
+1. Go to: **Settings > Printing > Print Settings > Raw Printing**.
+2. Check the `Enable Raw Printing` option.
+3. Save the settings.
```

Comment:
good steps. change step 4. to just "Save."

Reply:
Done.